mystyle.css

```diff
@@ -1,67 +1,306 @@
-body{
-    background-color: #fdfdfd;;
-}
-.main {
-    width: 500px;
-    margin: auto;
-}
-.container{
-    display: flex;
-   align-items: center;
-   flex-direction: row;
-   flex-wrap: wrap;
-   justify-content: space-between;
-}
-
-.navbar img{
-   
-    width: 30px;
-    height: 40px;
-}
-header img{
-    width: 180px;
-    height:120px;
-}
-.heading{
-    width:220px;
-}
-.content h1{
-    font-size: 70px;
-    font-weight: 400;
-    font-family: Playfair Display;
-}
-.para{
-    width: 340px;
-}
-.content{
-    display: flex;
-    align-items: center;
-    justify-content: center;
-    flex-direction: column;
-}
-p{
-    font-family: Poppins;
-    font-size: 30px;
-    font-weight: 400;
-
-}
-
-.rectangle{
-    background-color:#1A2236;
-    width: 288px;
-    padding: 20px;
-    color:white;
-    font-size: 24px;
-    font-family: Poppins, Sans Serif;
-    font-weight: 500;
-    padding-left: 80px;
-}
-.image{
-    display: flex;
-    align-items: center;
-    flex-direction: column;
-}
-.image img{
-    width: 350px;
-    margin-top: 60px;
-}
\ No newline at end of file
+body {
+  background-color: #fdfdfd;
+  margin:0;
+  padding:0;
+  min-width: 100vw;
+}
+.container {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+nav {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+  
+.navbar {
+  padding: 20px;
+}
+
+.navbar img {
+  width: 35px;
+}
+
+.content {
+  padding: 20px;
+}
+
+section {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+h1 {
+  font-family: 'Playfair Display', serif;
+  font-size: 50px;
+  font-weight: 400;
+  line-height: 57px;
+}
+
+.paragraph{
+  display: flex;
+}
+
+p {
```

Comment:
[Optional]
Kindly use CSS classes for styling the HTML elements As CSS classes are reusable and it's recommended as well. 👍🏻